frontend/src/plugins/api.js

```diff
@@ -281,11 +281,14 @@ class API {
 
     async login({username, password} = {}) {
         const url = `${this.base}/login`;
-        const body = `username=${username}&password=${password}`;
+        const body = new URLSearchParams({
+            username: username,
```

Comment:
Yes from testing it, it works (even with + in the password)...

But when I looked at the [docs](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams?retiredLocale=de#preserving_plus_signs), it suggested using `encodeURIComponent()`

When I look at the result it works and is correctly encoded, so no objections but still a bit confused by the docs